### PR TITLE
Connects #3931

### PR DESCRIPTION
**Pull request by Liew211**
Changed edit success message and new post message for team news.

### CODE REVIEW

File: src/app/news/news-list.component.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ import { DialogsPromptComponent } from '../shared/dialogs/dialogs-prompt.compone
 export class NewsListComponent {
 
   @Input() items: any[] = [];
+  @Input() editSuccessMessage: String = 'News has been updated successfully.';
```
Comment (lmmrssa) [inline src/app/news/news-list.component.ts]: it needs to be `string` not `String`

File: src/app/teams/teams-view.component.ts [hunk 2]
```diff
@@ -227,7 +227,7 @@ export class TeamsViewComponent implements OnInit, OnDestroy {
       viewableBy: 'teams',
       viewableId: this.teamId,
       ...message
-    }).pipe(finalize(() => this.dialogsLoadingService.stop())).subscribe(() => { this.dialogsFormService.closeDialogsForm(); });
+    }, 'Thank you for posting' ).pipe(finalize(() => this.dialogsLoadingService.stop())).subscribe(() => { this.dialogsFormService.closeDialogsForm(); });
```
Comment (lmmrssa) [inline src/app/teams/teams-view.component.ts]: It should be broken to two line. Codeclimate has issue due to long line of code

File: src/app/news/news-list.component.ts [hunk 3]
```diff
@@ -20,6 +20,7 @@ import { DialogsPromptComponent } from '../shared/dialogs/dialogs-prompt.compone
 export class NewsListComponent {
 
   @Input() items: any[] = [];
+  @Input() editSuccessMessage: string = 'News has been updated successfully.';
```
Comment (lmmrssa) [inline src/app/news/news-list.component.ts]: Codeclimate states not to declare type when value is assigned and value is self declarable.

**Comment by lmmrssa** [comment]
@Liew211 functionality is working good. Please fix issue from codeclimate

File: src/app/news/news-list.component.ts [hunk 4]
```diff
@@ -20,6 +20,7 @@ import { DialogsPromptComponent } from '../shared/dialogs/dialogs-prompt.compone
 export class NewsListComponent {
 
   @Input() items: any[] = [];
+  @Input() editSuccessMessage: 'News has been updated successfully.';
```
Comment (paulbert) [inline src/app/news/news-list.component.ts]: This isn't quite correct.  What you've done here is initialize a property that is undefined that is type limited to only accept the value `'News has been updated successfully.` in an assignment.

The line should be
```
@Input() editSuccessMessage = 'News has been updated successfully.';
```

Sorry I missed this last night, but when assigning a value to a variable we have a lint rule that rejects the code if there is a type assignment that matches the implied type based on the value of the variable.

File: src/app/news/news-list.component.ts [hunk 5]
```diff
@@ -46,7 +47,7 @@ export class NewsListComponent {
     this.dialogsFormService.openDialogsForm(title, fields, formGroup, {
       onSubmit: (response: any) => {
         if (response) {
-          this.newsService.postNews({ ...news, ...response }, 'News has been updated successfully.').subscribe(() => {
+          this.newsService.postNews({ ...news, ...response }, this.editSuccessMessage ).subscribe(() => {
```
Comment (paulbert) [inline src/app/news/news-list.component.ts]: You added an extra space after `editSuccessMessage`

File: src/app/teams/teams-view.component.ts [hunk 6]
```diff
@@ -227,7 +227,8 @@ export class TeamsViewComponent implements OnInit, OnDestroy {
       viewableBy: 'teams',
       viewableId: this.teamId,
       ...message
-    }).pipe(finalize(() => this.dialogsLoadingService.stop())).subscribe(() => { this.dialogsFormService.closeDialogsForm(); });
+    }, 'Message has been posted successfully' )
```
Comment (paulbert) [inline src/app/teams/teams-view.component.ts]: There's also an extra space here before the closing parenthesis.

**Pull request merged**